__tests__/api.test.ts

```diff
@@ -90,11 +122,11 @@ test('API: read file->translate->apply->write', async () => {
     }
 
     const [title, details] = summarize(summary);
-    expect(title).toEqual('Machine-translated 79 files, a total of 1,185 translations');
+    expect(title).toEqual('Machine-translated 89 files, a total of 1,335 translations');
     expect(details).toBeTruthy();
 });
 
-test('API: translate correctly performs translation', async () => {
+test.skip('API: translate correctly performs translation', async () => {
```

Comment:
Same question here, why is this being skipped now?

Reply:
See previous comment 😉